### PR TITLE
Middleware support for routes

### PR DESCRIPTION
**Pull request by tantam**
I have added middleware to configuration file and applied them to routes.
Please check it and modify what you think it is the best.
Thanks

### CODE REVIEW

File: config/l5-swagger.php [hunk 1]
```diff
@@ -82,6 +82,16 @@
 
     ],
 
+    /*
```
Comment (DarkaOnLine) [inline config/l5-swagger.php]: Can you move this to the routes array

File: config/l5-swagger.php [hunk 2]
```diff
@@ -82,6 +82,16 @@
...
+    |--------------------------------------------------------------------------
+    | Middleware allows to prevent unexpected access to API documentation
+    |--------------------------------------------------------------------------
+     */
+    'middleware' => [
+        'api' => ['web'],
```
Comment (DarkaOnLine) [inline config/l5-swagger.php]: What's the point to use `web` middleware ?

Comment (tantam) [inline config/l5-swagger.php]: this is middleware group from Laravel
https://laravel.com/docs/5.3/middleware#middleware-groups

File: config/l5-swagger.php (hunk 1 above)
Comment (tantam) [inline config/l5-swagger.php]: OK can

**Pull request merged**